tests/check_non_terminating.sh

```diff
@@ -7,6 +7,9 @@ DEVFILES_DIR="$(pwd)/stacks"
 # The stacks to test as a string separated by spaces
 STACKS=$(bash "$(pwd)/tests/get_stacks.sh")
 
+# Path to the check_non_terminating go package
+NON_TERMINATING_MODULE_BIN="$(pwd)/tests/check_non_terminating/./check_non_terminating"
```

Comment:
```suggestion
NON_TERMINATING_MODULE_BIN="$(pwd)/tests/check_non_terminating/check_non_terminating"
```

Do we need the `.` there?

Reply:
Also if the second `check_non_terminating` is the binary name maybe we rename it for clarity? So it isn't doubled up in the path

Reply:
I made it a bit more dynamic and gave it the name `flatten-parent` as default (also for the workflow).